src/StimulusBundle/assets/dist/loader.js

```diff
@@ -1,93 +1,98 @@
-import { Application } from '@hotwired/stimulus';
-import { isApplicationDebug, eagerControllers, lazyControllers } from './controllers.js';
-
-const controllerAttribute = 'data-controller';
-const loadControllers = (application, eagerControllers, lazyControllers) => {
-    for (const name in eagerControllers) {
-        registerController(name, eagerControllers[name], application);
-    }
-    const lazyControllerHandler = new StimulusLazyControllerHandler(application, lazyControllers);
-    lazyControllerHandler.start();
+import { Application } from "@hotwired/stimulus";
+import {
+  eagerControllers,
+  isApplicationDebug,
+  lazyControllers
+} from "./controllers.js";
```

Comment:
Thanks to `options.bundle = false` from ESBuild and `external` from tsup, the file `controllers.js` is **not** inlined, no BC here.

Reply:
Nice